src/Tizen.NUI.Components/Controls/NumberPicker.cs

```diff
@@ -0,0 +1,504 @@
+﻿/* Copyright (c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// NumberPickerValueChangedEventArgs is a class to notify changed NumberPicker value argument which will sent to user.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NumberPickerValueChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// NumberPickerValueChangedEventArgs default constructor.
+        /// <param name="value">value of NumberPicker.</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]   
+        public NumberPickerValueChangedEventArgs(int value)
+        {
+            Value = value;
+        }
+
+        /// <summary>
+        /// NumberPickerValueChangedEventArgs default constructor.
+        /// <returns>The current value of NumberPicker.</returns>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]   
+        public int Value { get; }
+        
+    }
+
+    /// <summary>
+    /// NumberPicker is a class which privides a function that allows the user to select 
+    /// a value through a scrolling mothion by expressing the specified value as a list.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NumberPicker : Control
+    {
+        //Tizen 6.5 base components Picker guide visibleScrollItem is 5.
+        private static int dummyItemsForLoop = 20; //Dummy item count for loop feature.
+        private static int itemHeight = 72; //Each list item height. Sibling with internal class's one.
```

Comment:
could you replace all the numbers as `const` if it is constant value?
and could replace them as meaning full named const member?

https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/general/g25-replace-magic-numbers-with-named-constants.html

for example, `private const int dummyItemsForLoop = 20;`  `private const int InitalItemHeight = 72; private static int itemHeight = InialItemHeight;` : these are just my idea, but if you intend another meanings, you can change in your own way.

Reply:
I agree with that. I changed static to const.